### ISSUE TITLE
v-b-tooltip error

### ISSUE THREAD

**Issue by tghkuma** [bootstrap-vue/bootstrap-vue-next#712]
**Describe the bug**

An error occurs in `v-b-tooltip` [beforeMount()](https://github.com/cdmoro/bootstrap-vue-3/blob/v0.3.5/packages/bootstrap-vue-3/src/directives/BTooltip.ts#L78) and [mount()](https://github.com/cdmoro/bootstrap-vue-3/blob/v0.3.5/packages/bootstrap-vue-3/src/directives/BTooltip.ts#L100) processing

**To Reproduce**

[https://stackblitz.com/edit/nuxt-starter-tiibpb?file=app.vue](https://stackblitz.com/edit/nuxt-starter-tiibpb?file=app.vue)
Check the JS console by clicking the button.

Delete other button comment and confirm.

**Expected behavior**
First, the following does not assume `bind.value=undefined`.
```
resolveTitle(binding.value).toString()
```

Then in the `update()` method, `title` is empty and stored in `data-bs-original-title`.

**Environment (please complete the following information):**

```
  "dependencies": {
    "@popperjs/core": "^2.11.6",
    "bootstrap": "^5.2.1",
    "bootstrap-vue-3": "^0.3.5"
  }
```

**Comment by devhus** [comment]
Bootstrap js scripts do not support SSR in case you are trying to use this library on NUXT project, it will not work correctly.

**Comment by tghkuma** [comment]
@devhus Of course `ssr=false`
https://stackblitz.com/edit/nuxt-starter-tiibpb?file=nuxt.config.ts

**Comment by VividLemon** [comment]
> Of course `ssr=false` https://stackblitz.com/edit/nuxt-starter-tiibpb?file=nuxt.config.ts

Would you be willing to test this on a basic Vue app? To determine if your issue is exactly Nuxt or exists in Vue as well

**Comment by tghkuma** [comment]
@VividLemon I tried with `Vue 3`
https://stackblitz.com/edit/vue-yw74js?file=src%2FApp.vue
had the same symptoms

After that, the following is no longer updated from the second button click. (same for nuxt3 version)
```
<div>{{ isEnabled }}:{{ titleDinamic }}</div>
```

**Comment by xvaara** [comment]
fixed in 0.8

without nuxt: https://stackblitz.com/edit/vue-o27vjt?file=src%2FApp.vue
I'm not sure what the first nuxt example is trying to do. But dynamic title is working in 0.8

@VividLemon close
